src/index.js

```diff
@@ -54,34 +54,42 @@ function matches (node, query) {
   return (node.matches || node.msMatchesSelector).call(node, query);
 }
 
-function nodeFromHtml (html) {
-  const div = document.createElement('div');
-  div.innerHTML = html;
-  return div.firstElementChild;
+function getInstantiatedNodeWithinFixture(node, isRootNode) {
+  const isStringNode = typeof node === 'string';
+
+  // If the fixture has been removed from the document, re-insert it.
+  if (!body.contains(fixture)) {
+    body.appendChild(fixture);
+  }
+
+  if (isRootNode) {
+    setFixtureContent( node, isStringNode );
+  }
+
+  return isStringNode
+    ? fixture.firstElementChild
+    : node;
+}
+
+function setFixtureContent( node, shouldSetChildrenViaString ) {
```

Comment:
Did semistandard not catch this? If not we should see why.

Function declaration should look like this: `function setFixtureContent (node, shouldSetChildrenViaString) {`